charts/timed/values.yaml

```
@@ -251,6 +251,12 @@ frontend:
     failureThreshold: 6
     # -- Minimum consecutive successes for the probe to be considered successful after having failed
     successThreshold: 1
+  settings:
+    sentry:
+      # -- Enable Sentry integration
+      enabled: false
+      # -- Sentry DSN for error reporting. Set to enable Sentry integration
+      dsn: ~
```

```suggestion
  settings:
      sentry:
      # -- Enable Sentry integration
      enabled: false
      # -- Sentry DSN for error reporting. Set to enable Sentry integration
      dsn: ~
      # -- Sentry trace sample rate, Set 1.0 to capture 100% of transactions
      tracesSampleRate: 1.0
      # -- Associate users to errors in Sentry
      sendDefaultPii: "True"
```

Please copy all the available settings from `backend`

---

@derrabauke implemented